jaeger-apachehttpclient/README.md

```diff
@@ -7,14 +7,11 @@ to instrument the client.
 On a high level, the RequestInterceptor starts a span and puts it into a HttpContext, and the ResponseInterceptor
 requests completes the span and reports it.
 
-As such, you will need to add both the RequestInterceptor and ResponseInterceptor to start seeing traces.
+To obtain an apache http client that already has the interceptors set, simply use `TracingHttpClients`
 
 # Example 
-
 ```
 Tracer tracer = jaegerConfig.getTracer();
-CloseableHttpAsyncClient client = HttpAsyncClients.custom()
-                                     .addInterceptorFirst(new JaegerRequestInterceptor(tracer))
-                                     .addInterceptorFirst(new JaegerResponseInterceptor())
-                                     .build();
+CloseableHttpAsyncClient client = TracingHttpClients.asyncBuilder(tracer).build();
```

Comment:
Could you elaborate on why accepting the builder as a parameter is better?
`HttpAsyncClientBuilder` and `HttpClientBuilder` don't have any common parents, nor do they implement any interfaces. 
We'd have something like the following instead of what we have right now.
```java
  public static HttpAsyncClientBuilder addTracer(HttpAsyncClientBuilder clientBuilder,
                                                 Tracer tracer) {
    return clientBuilder.addInterceptorFirst(new TracingRequestInterceptor(tracer))
                        .addInterceptorFirst(new TracingResponseInterceptor());
  }
  public static HttpClientBuilder addTracer(HttpClientBuilder clientBuilder,
                                                 Tracer tracer) {
    return clientBuilder.addInterceptorFirst(new TracingRequestInterceptor(tracer))
                        .addInterceptorFirst(new TracingResponseInterceptor());
  }
```